ql/src/test/org/apache/hadoop/hive/ql/exec/TestPartitionManagement.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.hadoop.hive.metastore;
+package org.apache.hadoop.hive.ql.exec;
```

Comment:
Moved TestPartitionManagement.java to ql module due to dependency on PartitionExpressionForMetastore and some other ql class for serializing partition expression.